OloEngine/src/OloEngine/Physics3D/JoltShapes.cpp

```
@@ -451,6 +455,82 @@ namespace OloEngine
         return shape != nullptr;
     }
 
+    JPH::Ref<JPH::Shape> JoltShapes::CreateTerrainHeightFieldShape(const std::vector<f32>& heights, u32 resolution,
```

Cppcheck (2.21.0)

[style] The function 'CreateTerrainHeightFieldShape' is never used.

(unusedFunction)